LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Actually we use another license, could you update it? (and also in package.json)